AppHandling/Run-AlPipeline.ps1

```diff
@@ -59,8 +59,14 @@
   Array or comma separated list of folders with apps to be compiled, signed and published
  .Parameter testFolders
   Array or comma separated list of folders with test apps to be compiled, published and run
+ .Parameter bcptTestFolders
+  Array or comma separated list of folders with bcpt test apps to be compiled, published and run
+ .Parameter pageScriptingTests
+  Array or comma separated list of filespecs with pageScripting tests, to be run after the apps have been compiled and tested
  .Parameter additionalCountries
   Array or comma separated list of countries to test
+ .Parameter restoreDatabases

```

Comment:
It is a request I had a while back, where I told people how to do this in a RunTests overrride - also Cosmo wanted to do a database snapshot (instead of a restore) - hence the override.
And if no restoreDatabases is specified - no time is used.